src/frontend/screens/Audio/AnimatedTimer.tsx

```diff
@@ -0,0 +1,50 @@
+import React, {FC} from 'react';
+import {StyleSheet, TextInput} from 'react-native';
+import Animated, {SharedValue, useAnimatedProps} from 'react-native-reanimated';
+import {WHITE} from '../../lib/styles';
+
+Animated.addWhitelistedNativeProps({text: true});
+const AnimatedTextInput = Animated.createAnimatedComponent(TextInput);
+
+interface AnimatedTimer {
+  elapsedTime: SharedValue<number>;
+}
+
+export function convertMsToTime(ms: number) {
```

Comment:
I don't like how it looks but for context: Bohdan had issues with passing functions from react-intl into reanimated UI thread, hence this method.